h/api/schemas.py

```diff
@@ -211,7 +211,7 @@ def validate(self, data):
             new_appstruct['shared'] = _shared(appstruct.pop('permissions'),
                                               new_appstruct['userid'])
         else:
-            new_appstruct['shared'] = True
+            new_appstruct['shared'] = False
```

Comment:
Hmm, maybe not. This only seems to be used in the `CreateAnnotationSchema` and not when updating.